data/plugins.json

```diff
@@ -1,10 +1,10 @@
 {
   "transform-es2015-arrow-functions": {
-    "chrome": 47,
+    "chrome": 45,
     "edge": 13,
     "firefox": 45,
     "safari": 10,
-    "node": 6
+    "node": 4
```

Comment:
Though I think that is the case with Node 6 too (but should be fixed in Node 7).